burn-wgpu/src/ops/float_ops.rs

```diff
@@ -139,6 +139,10 @@ where
         lhs: FloatTensor<Self, D>,
         rhs: FloatTensor<Self, D>,
     ) -> FloatTensor<Self, D> {
+        #[cfg(feature = "autotune")]
+        return kernel::matmul::tune::<G, F, D>(lhs, rhs);
```

Comment:
I think we should keep the autotune default, but only run if not in test mode.

`#[cfg(all(feature = "autotune", not(test))]`